app/harvesters/abstract_document_type_converter.py

```diff
@@ -0,0 +1,28 @@
+from abc import ABC
+
+from loguru import logger
+
+
+class AbstractDocumentTypeConverter(ABC):
+    """
+    Abstract mother class for all document type converters.
+    """
+
+    HARVESTER: str
+
+    TYPES_MAPPING: dict[str, tuple[str, str]]
+
+    UNKNOWN_CODE = ("http://data.crisalid.org/ref/document_types/unknown", "Unknown")
+
+    def __init__(self) -> None:
+        self.harvester = self.HARVESTER
+        self.types_mapping = self.TYPES_MAPPING
```

Comment:
I think I wouldn't have defined constants here. Also I'd let the necessary constant outside the class. I would have written it like this :  

```
UNKNOWN_CODE = ("http://data.crisalid.org/ref/document_types/unknown", "Unknown")

class AbstractDocumentTypeConverter(ABC):

    harvester: str

    types_mapping: dict[str, tuple[str, str]]
```